cli/src/main/java/bisq/cli/app/CommandParser.java

```diff
@@ -0,0 +1,27 @@
+package bisq.cli.app;
+
+import joptsimple.OptionParser;
+
+import static java.lang.System.err;
+
+final class CommandParser {
+
+    // Option name constants
+    static final String HELP = "help";
+    static final String GETBALANCE = "getbalance";
+    static final String GETVERSION = "getversion";
+    static final String STOPSERVER = "stopserver";
+
+    OptionParser configure() {
+        OptionParser parser = new OptionParser();
+        parser.allowsUnrecognizedOptions();
+        parser.nonOptions(GETBALANCE).ofType(String.class).describedAs("get btc balance");
+        parser.nonOptions(GETVERSION).ofType(String.class).describedAs("get bisq version");
+        return parser;
+    }
+
+    static void printUsage() {
+        err.println("Usage:  bisq-cli getbalance | getversion");
```

Comment:
Please study the way help output is printed in `BisqExecutable#execute` and note the use of `BisqHelpFormatter` which we may want to reuse (we would need to move it in order to do so, don't worry about it now, just use the default help formatter implementation in the meantime).

In any case, when help is explicitly requested, it shouldn't be printed to stderr, but stdout.

Please also explicitly `System.exit` 0 or 1 accordingly. You'll see how this is done in `BisqExecutable` as well.

Reply:
Prints usage message to stdout, and uses EXIT_SUCCESS and EXIT_FAILURE arguments in System.exit(). 

What's not done:  help formatting.  I need to discuss Bisq + `joptsimple.HelpFormatter` issues with @cbeams.  Right now, the only options passed to Cli are of type `NonOptionArgumentSpec`, which did not work as expected with `joptsimple.HelpFormatter`.  It looks like Bisq Config only uses `ArgumentAcceptingOptionSpec` options, which do work with `joptsimple.HelpFormatter`.  It did not seem correct to change non-option cli arguments to argument accepting options just to make them work with the `joptsimple.HelpFormatter` interface, which is why I checked in the

Reply:
Ok, that's fine then. We can come back to it later, thanks.